src/applications/vaos/appointment-list/redux/selectors.js

```diff
@@ -367,14 +369,10 @@ export function selectIsInPerson(appointment) {
 }
 
 export function selectPractitionerName(appointment) {
-  const { practitioners } = appointment;
+  const { providerName, practiceName, name } =
+    appointment.communityCareProvider || {};
 
-  if (!practitioners?.length) return null;
-
-  const practitioner = practitioners[0];
-  const { name } = practitioner;
-
-  return `${name?.given.toString().replaceAll(',', ' ')} ${name?.family}`;
+  return providerName || practiceName || name || null;
```

Comment:
the property returned for the provider name is coming from `practitioners` object not `communityCareProvider` because provider name is not exclusive to CC.